spec/cqm/converter/bonnie_measure_spec.rb

```diff
@@ -48,6 +46,17 @@
     expect(population_set.populations.IPP.hqmf_id).to eq('6CD39B4B-16E6-4FD4-9241-527F9F4A48D0')
     expect(population_set.populations.DENOM.statement_name).to eq('Denominator')
     expect(population_set.populations.DENOM.hqmf_id).to eq('B735F4BD-DFAE-45E3-BAA9-E09BDF731B8E')
+
+    # check source data criteria
```

Comment:
This block should be pulled out into a function as it's repeated below a few times.  Maybe name it `check_source_data_criteria` so the comment can be removed as well